### PR TITLE
Added Online Application Please Review this

### PR DESCRIPTION
**Pull request by jayavardhanvny**

### CODE REVIEW

File: OnlineApplication/.classpath [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (prasadram) [inline OnlineApplication/.classpath]: why this file is required

File: OnlineApplication/.project [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (prasadram) [inline OnlineApplication/.project]: why this file is required

File: OnlineApplication/.settings/.jsdtscope [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (prasadram) [inline OnlineApplication/.settings/.jsdtscope]: why this file is required

File: OnlineApplication/.settings/org.eclipse.jdt.core.prefs [hunk 4]
```diff
@@ -0,0 +1,7 @@
+eclipse.preferences.version=1
```
Comment (prasadram) [inline OnlineApplication/.settings/org.eclipse.jdt.core.prefs]: why this file is required

File: OnlineApplication/.settings/org.eclipse.wst.common.component [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?><project-modules id="moduleCoreId" project-version="1.5.0">
```
Comment (prasadram) [inline OnlineApplication/.settings/org.eclipse.wst.common.component]: why this file is required

File: OnlineApplication/.settings/org.eclipse.wst.common.project.facet.core.xml [hunk 6]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (prasadram) [inline OnlineApplication/.settings/org.eclipse.wst.common.project.facet.core.xml]: why this file is required

File: OnlineApplication/.settings/org.eclipse.wst.jsdt.ui.superType.container [hunk 7]
```diff
@@ -0,0 +1 @@
+org.eclipse.wst.jsdt.launching.baseBrowserLibrary
```
Comment (prasadram) [inline OnlineApplication/.settings/org.eclipse.wst.jsdt.ui.superType.container]: why this file is required

File: OnlineApplication/.settings/org.eclipse.wst.jsdt.ui.superType.name [hunk 8]
```diff
@@ -0,0 +1 @@
+Window
```
Comment (prasadram) [inline OnlineApplication/.settings/org.eclipse.wst.jsdt.ui.superType.name]: why this file is required

File: OnlineApplication/WebContent/WEB-INF/web.xml [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+    <welcome-file>index.html</welcome-file>
+    <welcome-file>index.htm</welcome-file>
+    <welcome-file>index.jsp</welcome-file>
+    <welcome-file>default.html</welcome-file>
+    <welcome-file>default.htm</welcome-file>
+    <welcome-file>default.jsp</welcome-file>
+  </welcome-file-list>
+  <servlet>
+    <description></description>
+    <display-name>TestServlet</display-name>
```
Comment (prasadram) [inline OnlineApplication/WebContent/WEB-INF/web.xml]: what is this TestServlet

File: OnlineApplication/WebContent/WEB-INF/web.xml [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+    <servlet-name>TestServlet</servlet-name>
+    <servlet-class>coreservlets.TestServlet</servlet-class>
+  </servlet>
+  <servlet-mapping>
+    <servlet-name>TestServlet</servlet-name>
+    <url-pattern>/TestServlet</url-pattern>
+  </servlet-mapping>
+  <servlet>
```
Comment (prasadram) [inline OnlineApplication/WebContent/WEB-INF/web.xml]: why it's empty

File: OnlineApplication/WebContent/WEB-INF/web.xml [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+    <description></description>
+    <display-name>NoInvokerServlet</display-name>
```
Comment (prasadram) [inline OnlineApplication/WebContent/WEB-INF/web.xml]: why you need this

File: OnlineApplication/WebContent/WEB-INF/web.xml [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+    <servlet-name>NoInvokerServlet</servlet-name>
+    <servlet-class>coreservlets.NoInvokerServlet</servlet-class>
+  </servlet>
+  <servlet-mapping>
+    <servlet-name>NoInvokerServlet</servlet-name>
+    <url-pattern>/NoInvokerServlet</url-pattern>
+  </servlet-mapping>
+  <servlet>
+    <description></description>
```
Comment (prasadram) [inline OnlineApplication/WebContent/WEB-INF/web.xml]: again testing ?

File: OnlineApplication/.classpath (hunk 1 above)
Comment (jayavardhanvny) [reply OnlineApplication/.classpath]: this file was created by eclipse....

File: OnlineApplication/src/jdbc/ConnectionJDBC.java [hunk 13]
```diff
@@ -0,0 +1,55 @@
+package jdbc;
```
Comment (prasadram) [inline OnlineApplication/src/jdbc/ConnectionJDBC.java:1]: please follow naming convention

File: OnlineApplication/src/jdbc/ConnectionJDBC.java [hunk 14]
```diff
@@ -0,0 +1,55 @@
...
+		
+		
+		prop.setProperty("user", "postgres");
+		prop.setProperty("password", "password");
+		prop.setProperty("ssl", "true");
+		Connection connection = null;
+		//String url = "jdbc:postgresql://localhost:5432;"+"databaseNamepostgres;", "postgres", "password";
+		try{
+			
+			String url = "jdbc:postgresql://localhost:5432/"+database;
```
Comment (prasadram) [inline OnlineApplication/src/jdbc/ConnectionJDBC.java:42]: why these are hardcoded

**Review by prasadram — CHANGES_REQUESTED** [review]
Please don't push binary files

**Comment by prasadram** [comment]
raise a new pr

**Pull request closed without merging**